### DISCUSSION TITLE
Device Spectral Sensor Internal Error

### DISCUSSION THREAD

**Discussion by Ahmad-197** [cubert-hyperspectral/cuvis.sdk#58] [Q&A] [answered]
I am new to using Ultris X50 Plus camera. I am unable to use my camera using Cubert Utlis Touch and Cubert Utlis Core. I receive the following error:

![image](https://github.com/cubert-hyperspectral/cuvis.sdk/assets/86935871/fefe7ec8-7fce-45cf-ac5f-9b5fe27699d4)

Can anyone tell me what might be the problem and how to resolve this issue?

**Comment by robertCubert** [comment]
Hi Ahmad, 

as this is not related to the SDK please open a service ticket via our [helpdesk ](https://cuvis.atlassian.net/servicedesk/customer/portal/3)and we will help you out with your issues!

**Comment by Ahmad-197** [comment, marked as answer]
> I am new to using Ultris X50 Plus camera. I am unable to use my camera using Cubert Utlis Touch and Cubert Utlis Core. I receive the following error:
> 
> ![image](https://private-user-images.githubusercontent.com/86935871/310880361-fefe7ec8-7fce-45cf-ac5f-9b5fe27699d4.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTIxNTQzOTgsIm5iZiI6MTcxMjE1NDA5OCwicGF0aCI6Ii84NjkzNTg3MS8zMTA4ODAzNjEtZmVmZTdlYzgtN2ZjZS00NWNmLWFjNWYtOWI1ZmUyNzY5OWQ0LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MDMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDAzVDE0MjEzOFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWIyMjZjOTFkYzQ2ZmJmNGI4NDA1MzlhOGI4MzZmMDBjNGE2MjliMDg4NDY5ZTY3ZWZkOWM3NDJjOGNhMjE1MWEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.seSlWaj6zLisM6BjKywR_TmkU1hicsr7RRTAxTbP9vI)
> 
> Can anyone tell me what might be the problem and how to resolve this issue?

The issue was related to the ip addresses, by allocating proper ip addresses and reinstalling the factory settings the issue was resolved.

**Reply by arndcubert** [reply]
Dear Ahmad,
I am sorry to hear that you have problems with your camera. Please state your case in the service portal: http://support.cubert-hyperspectral.com/ 
Kind regards
Arnd
